docs/api/modules/Internationalization.md

```diff
@@ -63,26 +65,32 @@ Example `en-US.json`:
 
 ### Environment Specific Data
 
-[one-app-locale-bundler](https://github.com/americanexpress/one-app-cli/tree/master/packages/one-app-locale-bundler) also allows your module to provide environment
+[`one-app-locale-bundler`](https://github.com/americanexpress/one-app-cli/tree/master/packages/one-app-locale-bundler) also allows your module to provide environment
 specific data. Currently One App supports three different environments;
 production, qa and integration (development).
 
-Below is an example of how you could include link specific data.
+The directory name is what defines the key the data goes under,
+below is an example of how you could include "link" specific data.
+
 
 Module Root:
 ```
 module
-└── locale
-| └── en-US
-|     └── links
-|     |   └── production.json
-|     |   └── qa.json
-|     |   └── integration.json
-|     └── copy.json
-| └── es-MX.json
-|── src
-|   └── index.js
-└── package.json
+├── README.md
+├── locale
+|   └── en-US
+|   |   ├── links
+|   |   |   └── production.json
+|   |   |   └── qa.json
+|   |   |   └── integration.json
+|   |   └── copy.json
+|   └── es-mx.json
```

Comment:
this shouldnt be `.json` since its a dir, its also missing the links dir and `copy.json`